src/pages/ItemList/components/Product.js

```diff
@@ -0,0 +1,22 @@
+import { BsCart3 } from 'react-icons/bs';
+import './product.scss';
+
+export const Product = ({ name, thumbnail, price, descriptions }) => {
+  return (
+    <article className="product">
+      <div
+        className="productImage"
+        style={{
+          backgroundImage: `url(${thumbnail})`,
+        }}
+      >
+        <div className="cartIcon">
+          <BsCart3 className="cart" size="25px" />
+        </div>
+      </div>
```

Comment:
```suggestion
<div className="productCard">
        <img className="productImage" src={thumbnail} alt="product" />
        <div className="cartIcon">
          <BsCart3 className="cart" size="25px" />
        </div>
      </div>
```
이렇게 설정하고, 이미 productCard에 적용된 style인 
positionSort에 default로 position:absolute가 있어서 구현이 가능해지네요!